products/coder.md

```diff
@@ -0,0 +1,80 @@
+title: Coder
+category: server-app
+tags: remote cde terraform ide development docker kubernetes
+iconSlug: coder
+permalink: /coder
+versionCommand: coder version
+releasePolicyLink: https://coder.com/docs/install/releases#release-schedule
+changelogTemplate: https://github.com/coder/coder/releases/tag/v__LATEST__
+releaseDateColumn: true
+eoasColumn: Mainline
+eoasWarnThreshold: 30
+eolColumn: Stable
+eolWarnThreshold: 60
+eoesColumn: Security Support
+eoasWarnThreshold: 150
+
+auto:
+  methods:
+  - git: https://github.com/coder/coder.git
+
+releases:
+- releaseCycle: "2.13"
+  releaseDate: 2024-07-02
+  eoas: 2024-08-06
+  eol: 2024-09-03
+  eoes: false
+  latest: "2.13.1"
+  latestReleaseDate: 2024-07-20
```

Comment:
```suggestion
  latest: "2.13.2"
  latestReleaseDate: 2024-07-23
```

Reply:
Thank you. I wanted to bump this version.